docs/cli/tkgctl/management-cluster-using-clusterclass.md

```diff
@@ -1,20 +1,23 @@
 # Management Cluster Deployment using ClusterClass
 
-Below is the sequence of management-cluster creation using clusterclass.
+Below is the sequence of steps involved in creating a management cluster ClusterClass.
 
 1. Deploy a bootstrap kind cluster
 1. Deploy `kapp-controller` on bootstrap cluster
 1. Deploy `providers` on bootstrap cluster
-1. Deploy `TKG meta package` to bootstrap cluster
+1. Deploy `tanzu-framework package` to bootstrap cluster
+1. Deploy `tkg composite package` to bootstrap cluster
```

Comment:
I would like to see descriptions of these packages.

Reply:
Ack, will provide some more information on those.

Reply:
Updated PR to mention tkg (composite) and framework packages and their relationships. The term 'composite' as I have seen mentioned was referring to fact that it is used to compose and drive the installation of a collection of other related sub-packages 
Deferring to @anujc25 to clarify the terminology we have been using.

Reply:
Done